src/core_plugins/kibana/public/dashboard/listing/dashboard_landing_page_table.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { LandingPageTable } from 'ui_framework/components';
+import { getDashboardColumns } from './get_dashboard_columns';
+import { ItemTableActions } from 'ui/saved_object_table/item_table_actions';
+import { DashboardConstants } from '../dashboard_constants';
+
+export function DashboardLandingPageTable({ itemTableState, onDeleteItems, onFilterItems, isFetchingItems }) {
+  const columns = getDashboardColumns(itemTableState);
+
+  return <LandingPageTable
```

Comment:
(Yellow flag) This abstraction seems a bit too monolithic because of the massive number of props. This is analogous to a function signature with 12 arguments.